### PR TITLE
Handle log level

### PR DESCRIPTION
**Pull request by he2ss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,8 @@ The configuration is located by default in `/usr/local/lua/crowdsec/crowdsec.con
 ```
 API_URL=http://localhost:8080                 <-- the API url
 API_KEY=                                      <-- the API Key generated with `cscli bouncers add -n <bouncer_name>` 
-LOG_FILE=/tmp/lua_mod.log                     <-- path to log file
+LOG_FILE=/tmp/lua_mod.log                     <-- path to log file
```
Comment (blotus) [inline README.md:54]: We should default to a better path.

File: lib/config.lua [hunk 2]
```diff
@@ -35,7 +35,7 @@ function config.loadConfig(file)
         return nil, "File ".. file .." doesn't exist"
     end
     local conf = {}
-    local valid_params = {'API_URL', 'API_KEY', 'LOG_FILE'}
+    local valid_params = {'API_URL', 'API_KEY', 'LOG_FILE', 'LOG_LEVEL'}
```
Comment (blotus) [inline lib/config.lua:38]: Do we want to add an explicit default value in the code ?

Comment (buixor) [reply lib/config.lua:38]: the `getLogLevel` returns `logging.INFO` by defaut, might do the trick ?

Comment (he2ss) [reply lib/config.lua:38]: Yep, but I also added default value for the `LOG_FILE` and `LOG_LEVEL` :+1:

**Pull request merged**